modules/tinymce/src/core/test/ts/browser/DragDropOverridesTest.ts

```diff
@@ -194,6 +280,189 @@ describe('browser.tinymce.core.DragDropOverridesTest', () => {
       await pAssertNotification('Dropped file type is not supported');
     });
 
+    it('TINY-9601: dataTransfer data modified in dragstart event should persist in drop and dragend events and in content inserted on drop', async () => {
+      const editor = hook.editor();
+
+      const testImage = {
```

Comment:
Is there a type we could use for these if not then add one just to the test. Like TestImage or something.

Reply:
Exposed DragImageData type from dragster to be used here.